content/en/apps/features/integrations/generic.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Generic"
+weight: 1
+description: >
+   Integrate with any system using RESTful APIs
+keywords: generic
+relatedContent: >
+  apps/features/integrations/dhis2
+  apps/features/integrations/rapidpro
+  apps/features/integrations/openmrs
+---
+
+The CHT Core Framework includes functionality that allows you to share data with any API-based system. App Builders have configured CHT integrations with OpenMRS, KenyaEMR, Bahmni, DHIS2, RapidPro, Apache NiFi, OpenHIM, custom EMRs, and several other systems.  
+
+## Overview
+
+Integrating a CHT App into your digital health ecosystem starts with identifying an integration use case. It's important to first understand all the components present in the ecosystem (EMR, laboratory system, community health information system, etc) and then plan out what the workflow will look like operationally. It is important to consider what information is needed at each point, will it be available to them, what happens if it is not, is this workflow even useful for them.
```

Comment:
Maybe a bulleted list would work better here (for the 4 items at the end of the paragraph)?

Reply:
Since this is broadly focused on integrations, it would be helpful to set the stage a bit further:

*Integrations definition and how it applies in our context- something around unifying multiple different information systems, making electronic data available to users across multiple platforms, harmonizing data

*CHT's architecture- designed to receive and host data from different data sources + share it across systems